eng/updateversions.proj

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+  <PropertyGroup>
+    <ArcadeBuild>true</ArcadeBuild>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <Import Project="$(NuGetPackageRoot)\microsoft.dotnet.versiontools.tasks\$(MicrosoftDotNetVersionToolsTasksPackageVersion)\build\Microsoft.DotNet.VersionTools.Tasks.targets" />
```

Comment:
Technically this is the only thing that we still have from buildtools?

Reply:
This now comes from arcade: https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.VersionTools :)